site/pages/rest-api/api/site.md

```diff
@@ -331,13 +330,11 @@ You can also concatenate the two options as follows:
 {% restapi "site/{site type}/{site name}/file/{file name}", "POST", "Creates a site file with the specified name." %}
 The parameter "isClientFile" with value 1 can be set in the request to send the file to clients.
 
-**Note:** This method is applicable only to IBM Enpdoint Manager V9.2 and later versions. 
```

Comment:
I would prefer leaving unchanged documentation when related to obsoleted versions, to keep support for legacy implementations and environments